src/Symfony/Component/Scheduler/Schedule.php

```diff
@@ -141,20 +141,32 @@ public function getSchedule(): static
 
     public function before(callable $listener, int $priority = 0): static
     {
+        if (!$this->dispatcher) {
+            throw new LogicException(sprintf('To register a "%s" listener, you need to set an event dispatcher on the Schedule.', __METHOD__));
```

Comment:
Maybe reword this a bit?

```suggestion
            throw new LogicException(sprintf('To register a listener with "%s()" you need to set an event dispatcher on the Schedule.', __METHOD__));
```

Reply:
LGTM